ocs_ci/ocs/ui/views.py

```diff
@@ -2024,7 +2024,65 @@
     "alerting_rule_details_link": ("//a[normalize-space()='{}']", By.XPATH),
 }
 
-
+bucket_tab = {
+    "create_bucket_button_obc": (
+        "//div[text()='Create via Object Bucket Claim']",
+        By.XPATH,
+    ),
+    "storage_class_dropdown": (
+        "//button[contains(., 'SCSCSelect StorageClass')]",
+        By.XPATH,
+    ),
+    "storage_class_noobaa_option": (
+        "#openshift-storage\\.noobaa\\.io-link > a > div:nth-child(1) > span",
```

Comment:
this will fail if storage namespace is not "openshift-storage" 
we'll need to use config.ENV_DATA["cluster_namespace] 
also: are you sure about index, that it will be always [1] ? here > `div:nth-child(1)`